RecoEgamma/ElectronIdentification/src/ElectronMVAEstimatorRun2Fall17.cc

```diff
@@ -0,0 +1,428 @@
+                                             spp,                      // 1
+                                             OneMinusE1x5E5x5,
+                                             R9,
+                                             etawidth,
+                                             phiwidth,                 // 5
+                                             HoE,
+                                             //Pure tracking variables
+                                             kfhits,
+                                             kfchi2,
+                                             gsfchi2,                  // 9
+                                             // Energy matching
+                                             fbrem,
+                                             gsfhits,
+                                             expectedMissingInnerHits,
+                                             convVtxFitProbability,     // 13
+                                             EoP,
+                                             eleEoPout,                // 15
+                                             IoEmIoP,
+                                             // Geometrical matchings
+                                             deta,                     // 17
+                                             dphi,
+                                             detacalo,
+                                             // Isolation variables
+                                             ele_pfPhotonIso,
+                                             ele_pfChargedHadIso,
+                                             ele_pfNeutralHadIso,
+                                             // Pileup
+                                             (float)*rho,
+
+                                             // Endcap only variables
+                                             // NOTE: we don't need to check if
+                                             // we are actually in the endcap
+                                             // or not, as it is the last
+                                             // variable in the vector and it
+                                             // will be ignored by the
+                                             // GBRForest for the barrel.
+                                             // The GBRForest classification
+                                             // just needs an array with the
+                                             // input variables in the right
+                                             // order, what comes after doesn't
+                                             // matter.
+                                             PreShowerOverRaw          // 24
+                                        )
+                        );
+
+    constrainMVAVariables(vars);
+
+    return vars;
+  }
+  else
+  {
+    std::vector<float> vars = std::move( packMVAVariables(
```

Comment:
he std::move() is unnecessary its already an rvalue. Regardless, having looked at packMVAVariables (it just makes a vector) it would be better to just do
std::vector vars {
instead as it would achieve exactly the same thing and is much more readable